basic_pipeline/03_Source.md

```diff
@@ -1,7 +1,7 @@
 # Source
 
 Let's get to the code!
-We will start where all the [pipelines](../glossary/glossary.md#pipeline) start - with the `Source` [element](../glossary/glossary.md/#element).
+We will start where all the [pipelines](../glossary/glossary.md#pipeline) start - with the `Source` [element](../glossary/glossary.md#element).
```

Comment:
We should consider linking to Element.Source in that place - but I am not 100% sure about that, perhaps we should give a wider outlook on the different element types and link to the `element`.